config.go

```diff
@@ -7,9 +7,89 @@ import (
 	"time"
 )
 
+// These are the versions of the protocol (which includes RPC messages as
+// well as Raft-specific log entries) that this server can _understand_. Use
+// the ProtocolVersion member of the Config object to control the version of
+// the protocol to use when _speaking_ to other servers. This is not currently
+// written into snapshots so they are unversioned. Note that depending on the
```

Comment:
Fixed this over in Consul by adding some code to blow away the peers.json file on the first boot - https://github.com/hashicorp/consul/pull/2222/commits/82084c2a6c8a8e5a3dced3bcb71db4f0d4dba278.